libs/banglejs/jswrap_bangle.c

```diff
@@ -5904,6 +5904,8 @@ On Bangle.js there are a few additions over the standard `graphical_menu`:
     menu is removed
   * (Bangle.js 2) `scroll : int` - an integer specifying how much the initial
     menu should be scrolled by
+  * `onchange : function(value, touch) {}` - on Bangle.js 2 this function can
```

Comment:
I don't think this is in the right place - I think it means the `"":{...}` object here?

Reply:
Aha - I see what you mean.